package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "stitch-js-sdk",
+	"name": "stitch-js-sdk",
```

Comment:
[nit] I can't tell if these are tabs or spaces from the Github interface, but make sure it's consistent across the whole file.